.github/workflows/post-atlas-operator-release.yaml

```diff
@@ -0,0 +1,100 @@
+name: Create PR with Atlas Operator Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'version'
+        required: true
+
+jobs:
+  create-pr:
+    permissions: write-all
+    name: Create Release branch
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone repositories
+        run: |
+          mkdir operator
+          cd operator
+          git clone https://github.com/mongodb/mongodb-atlas-kubernetes.git
+          cd ..
+          mkdir helm
+          cd helm
+          git clone https://github.com/Sugar-pack/helm-charts.git
```

Comment:
Sorry for jumping late, but is it good to use your personal repo here?
Why not clone the MongoDB repo using the checkout action instead of the fork?

Reply:
You are right. I just forgot to fix this line.

Reply:
Fixed https://github.com/mongodb/helm-charts/pull/188